### PR TITLE
feat:build cart page

### PR DESCRIPTION
**Pull request by ishitakeshawani**
Preview link: https://deploy-preview-3--stunnershub.netlify.app/pages/cart/cart.html

### CODE REVIEW

File: pages/cart/cart.css [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+  justify-content: center;
+  background: var(--white);
+  box-shadow: 0px 2px 3px 0 rgb(0 0 0 / 10%);
+}
+
+.cart-product-details {
+  display: flex;
+  flex-direction: column;
+  padding: 1rem;
+  background: var(--);
```
Comment (anupkgurung) [inline pages/cart/cart.css]: no variable value ?

File: pages/cart/cart.css [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+.flex-row {
+  display: flex;
+  justify-content: space-between;
+}
+
+.footer-items {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
```
Comment (anupkgurung) [inline pages/cart/cart.css:134]: (While Refactoring) you can club flex property in one class and use that as direction column and center align is used.

File: pages/products/products.css [hunk 3]
```diff
@@ -53,7 +53,9 @@ body {
 }
 
 .sort-by-title {
-  margin: 1rem 0 1rem 0;
+  margin-top: 1rem;
+  margin-bottom: 1rem;
+  margin: 1rem 0 1rem;
```
Comment (anupkgurung) [inline pages/products/products.css]: why multiple margin values margin top bottom than margin alone ?

**Review by anupkgurung — COMMENTED** [review]
left some comments.

File: pages/cart/cart.css (hunk 1 above)
Comment (ishitakeshawani) [reply pages/cart/cart.css]: forgot to remove

File: pages/cart/cart.css (hunk 2 above)
Comment (ishitakeshawani) [reply pages/cart/cart.css:134]: yes

File: pages/products/products.css (hunk 3 above)
Comment (ishitakeshawani) [reply pages/products/products.css]: forgot to remove margin top and bottom

**Review by anupkgurung — COMMENTED** [review]
lgtm

**Pull request merged**